File: 1805 - A+B Complejo/problem_1805.cs
using System;

namespace problem_1805
{
    class Program
    {
        static void Main(string[] args)
        {
            String cad;
            long A, B, res;
            String [] a = new String[2];
            cad = Console.ReadLine();
            a = cad.Split(' ');
            A = Convert.ToInt64(a[0]);
            B = Convert.ToInt64(a[1]);
            res = 2*(A+B);
            Console.WriteLine(res);
        }
    }
}

File: 1805 - A+B Complejo/problem_1805.java
import java.util.Scanner;


public class problem_1805 {
    public static void main(String[] args) {
        Scanner e = new Scanner(System.in);
        String cad;
        long A,B, res;
        String[] a = new String[2];
        cad = e.nextLine();
        a = cad.split(" ");
        A = Integer.parseInt(a[0]);
        B = Integer.parseInt(a[1]);
        res = 2*(A+B);
        System.out.println(res);
    }
}
